module1/projects/connect_four/iteration_2.md

```diff
@@ -10,24 +10,148 @@ _[Back to Requirements](./requirements)_
 
 In this iteration, you are required to use TDD to create your classes. Use the interaction pattern to determine what a method should do and write one or more tests to verify that expected behavior. Then you can implement the method. You should always write code with the purpose of making a test pass.
 
-## Placing Pieces
+## Board
 
-Update your program to request user input and allow them to place an individual piece.
+The Board class is responsible for keeping track of each cell on the board and validating 4 in a row. 
 
-Your program should ask the user to enter a letter A-G.
-Update the board to display the player's piece in the lowest available position of the selected column with an 'X'.
+Follow the interaction pattern below to build out the functionality for placing a chip on the board:
 
-The computer takes its turn by selecting one of the 7 columns at random. Update the board to display the computer's piece in the lowest available position of the selected column with an 'O'.
 
-The player and computer should be able to repeat this sequence and continue taking turns.
+### Place Chip On Board
 
-### Invalid Column Selection
+```ruby
+pry(main)> require './lib/cell'
+# => true
+pry(main)> require './lib/board'
+# => true
+pry(main)> a_cells = [Cell.new("A1"),Cell.new("A2"),Cell.new("A3"),Cell.new("A4"),Cell.new("A5"),Cell.new("A6")]
+pry(main)> b_cells = [Cell.new("B1"),Cell.new("B2"),Cell.new("B3"),Cell.new("B4"),Cell.new("B5"),Cell.new("B6")]
+pry(main)> c_cells = [Cell.new("C1"),Cell.new("C2"),Cell.new("C3"),Cell.new("C4"),Cell.new("C5"),Cell.new("C6")]
+pry(main)> d_cells = [Cell.new("D1"),Cell.new("D2"),Cell.new("D3"),Cell.new("D4"),Cell.new("D5"),Cell.new("D6")]
+pry(main)> e_cells = [Cell.new("E1"),Cell.new("E2"),Cell.new("E3"),Cell.new("E4"),Cell.new("E5"),Cell.new("E6")]
+pry(main)> f_cells = [Cell.new("F1"),Cell.new("F2"),Cell.new("F3"),Cell.new("F4"),Cell.new("F5"),Cell.new("F6")]
+pry(main)> g_cells = [Cell.new("G1"),Cell.new("G2"),Cell.new("G3"),Cell.new("G4"),Cell.new("G5"),Cell.new("G6")]
+pry(main)> columns = {A: a_cells, B: b_cells, C: c_cells, D: d_cells, E: e_cells, F: f_cells, G: g_cells}
+pry(main)> board = Board.new(columns)
+pry(main)> board.place_chip("A", "X")
+pry(main)> board.render_board
```

Comment:
I think this and the following `board.render_board` calls are meant to be `board.render` to match I1.